### PR TITLE
Improve printable index

### PR DESCRIPTION
**Pull request by DarthGandalf**

### CODE REVIEW

**Review by qwrtln — CHANGES_REQUESTED** [review]
Doesn't work in the workflow.

**Comment by qwrtln** [comment]
This is the index page from the PDF built in the workflow:

en-55

File: .github/workflows/build-and-publish-rule-book.yaml [hunk 1]
```diff
@@ -133,13 +134,9 @@ jobs:
         uses: xu-cheng/latex-action@v3
         with:
           extra_system_packages: "inkscape"
+          pre_compile: export HOMM3_PRINTABLE=1 HOMM3_LANG=${{ matrix.language }}
```
Comment (qwrtln) [inline .github/workflows/build-and-publish-rule-book.yaml:137]:
```suggestion
        uses: xu-cheng/latex-action@v3
        env:
          HOMM3_PRINTABLE: 1
          HOMM3_LANG: ${{ matrix.language }}
        with:
          extra_system_packages: "inkscape"
```

Comment (qwrtln) [reply .github/workflows/build-and-publish-rule-book.yaml:137]: Not sure if it works on actions, though

Comment (qwrtln) [reply .github/workflows/build-and-publish-rule-book.yaml:137]: If that doesn't work, I guess you can just do 
```
echo "HOMM3_PRINTABLE=1" >> $GITHUB_ENV
```
in a previous step.

**Comment by DarthGandalf** [comment]
Fixed the empty index.

Caveat: I simply took the first character, and compared their codepoints, ignoring existence of collation, so for some languages the result may be wrong. This will be easier to fix when such cases appear

**Comment by qwrtln** [comment]
Ukrainian index doesn't look good. Does it work for Cyrillic alphabet?

image

**Review by qwrtln — APPROVED** [review]
Perfect

**Pull request merged**